src/viur/core/bones/user.py

```diff
@@ -1,31 +1,49 @@
+import typing as t
 from viur.core import current
 from viur.core.bones.relational import RelationalBone
 
 
 class UserBone(RelationalBone):
     """
     A specialized relational bone for handling user references. Extends the functionality of
-    :class:`viur.core.bones.relational.RelationalBone` to include support for creation and update magic.
-
-    :param bool creationMagic: If True, the bone will automatically store the creating user when a new entry is added.
-    :param bool updateMagic: If True, the bone will automatically store the last user who updated the entry.
-    :param bool visible: If True, the bone will be visible in the skeleton.
-    :param bool readOnly: If True, the bone will be read-only and its value cannot be changed through user input.
-    :param dict kwargs: Additional keyword arguments passed to the parent class constructor.
-    :raises ValueError: If the bone has multiple set to True and a creation/update magic is set.
+    :class:`viur.core.bones.relational.RelationalBone` to include support for creation and update magic,
+    and comes with a predefined descr, format, kind and refKeys setting.
+
+    :param creationMagic: If True, the bone will automatically store the creating user when a new entry is added.
+    :param updateMagic: If True, the bone will automatically store the last user who updated the entry.
+
+    :raises ValueError: If the bone is multiple=True and creation/update magic is set.
```

Comment:
This would fit better in `__init__`'s (not yet existing) docstring.

Reply:
Moved in [02d278a](https://github.com/viur-framework/viur-core/pull/1055/commits/02d278aa27aaca6456039180e69bd1470604230f)